src/groupeddatatable/grouping.jl

```diff
@@ -236,12 +236,28 @@ colwise(sum, groupby(dt, :a))
 ```
 
 """
-colwise(f::Function, d::AbstractDataTable) = map(i -> f(d[i]), 1:ncol(d))
+function colwise(f::Function, d::AbstractDataTable)
+    x = [f(d[i]) for i in 1:ncol(d)]
+    if eltype(x) <: Nullable && isa(x, Vector{eltype(x)})
```

Comment:
I wanted to distinguish between `Array{Nullable}` & `NullableArray` as they both have `eltype == Nullable`, but I just checked and converting a `NullableArray` to a `NullableArray` is a no-op so you're right, we can remove that check and it shouldn't hurt anything